modules/adagioAnalyticsAdapter.js

```diff
@@ -221,6 +225,7 @@ function handlerAuctionInit(event) {
       return request ? request.bids[0].src : null
     }
     const biddersSrc = sortedBidderCodes.map(bidSrcMapper).join(',');
+    const biddersCodes = sortedBidderCodes.map(bidder => getAdapterNameForAlias(bidder)).join(',');
```

Comment:
`biddersCodes` is correct but in this case, I would like to explicitly understand what we have.
I'd prefer something like `biddersOgCode`, `biddersRealCode`, `biddersRawCode` (singular to keep consistency with `bdrs_code` and `biddersSrc` above)

Or feel free to rename `sortedBidderCodes` if you think it is better

Reply:
Finally I preferred to rename `sortedBidderCodes` into `sortedBidderNames` and to keep the name `bidderCode` which has more sense in the context.

Reply:
Agree